assets/css/main.scss

```diff
@@ -1,8 +1,139 @@
 ---
 ---
+// This file should not be modified unless by an advanced user.
 
-
-// U.S. Web Design System setup 
+// U.S. Web Design System setup
 $font-path: '../uswds/fonts';
 $image-path: '../uswds/img';
+// U.S. Web Design System variable setup
+@import 'uswds/lib/bourbon';
+@import 'uswds/lib/neat';
+@import 'uswds/lib/normalize';
+@import 'uswds/core/variables';
+
+// Setting typography variables
+{% if site.data.theme.fonts %}
+  // Typography
+  {% if site.data.theme.fonts.sans %}$font-sans: {{ site.data.theme.fonts.sans | default: "$font-sans" }};{% endif %}
+  {% if site.data.theme.fonts.serif %}$font-serif: {{ site.data.theme.fonts.serif | default: "$font-serif" }};{% endif %}
+  {% if site.data.theme.fonts.normal %}$font-normal: {{ site.data.theme.fonts.normal | default: "$font-normal" }};{% endif %}
+  {% if site.data.theme.fonts.bold %}$font-bold: {{ site.data.theme.fonts.bold | default: "$font-bold" }};{% endif %}
+{% endif %}
+
+// Setting color variables
+{% if site.data.theme.colors %}
```

Comment:
Given that there are defaults defined for each variable now, I don't think the extra conditional statement wrapping each section are necessary. Would you mind if I took a stab at removing them and see if everything still works as expected?